addons/website_slides/views/rating_rating_views.xml

```diff
@@ -99,14 +97,72 @@
         </field>
     </record>
 
+    <record id="rating_review_view_kanban_slide_channel" model="ir.ui.view">
```

Comment:
Give it a greater priority so that it is not a default view but a real custom view used in a specific action (lower priority wins).